### ISSUE TITLE
Standard-DeepRL agents

### ISSUE THREAD

**Issue by MathisFederico** [MathisFederico/LearnRL#3]
In need if issue #1 : Agents

# Standard-DeepRL agents

## Deep-Qlearning
- Need DataStoring for experience replay

## Actor-Critic
- Need Tensorflow/Pytorch for custom gradient descent
